packages/gravity-ui/package.json

```diff
@@ -0,0 +1,99 @@
+{
+  "name": "@buildit/gravity-ui-web",
+  "version": "1.0.1",
+  "description": "Library of styles, components and associated assets to build UIs for the web. Part of Buildit's Gravity design system.",
+  "main": "dist/ui-lib/gravity.js",
+  "files": [
+    "build-api.js",
+    "build-consts.js",
+    "dist/ui-lib/",
+    "src/ui-lib/",
+    "eyeglass-exports.js"
+  ],
+  "scripts": {
+    "clean": "gulp clean",
+    "build": "gulp",
+    "styleguide": "gulp styleguide",
+    "start": "gulp serve",
+    "test": "npm run lint",
+    "lint": "stylelint 'src/ui-lib/sass/**/*.scss'",
+    "semantic-release": "semantic-release"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/buildit/gravity-ui-web.git"
+  },
+  "keywords": [
+    "eyeglass-module",
+    "sass",
+    "ui",
+    "css",
+    "web",
+    "design",
+    "gravity",
+    "buildit",
+    "eyeglass",
+    "style",
+    "design-system",
+    "style-guide"
+  ],
+  "author": {
+    "name": "Buildit",
+    "url": "http://buildit.wiprodigital.com/"
+  },
+  "contributors": [
+    "Matteo Pescarin (http://peach.smartart.it)",
+    "Ben Oakenfull",
+    "Marcos Peebles",
+    "James Nash (http://cirrus.twiddles.com/)",
+    "Valerio Francescangeli",
+    "Daniel Bull"
```

Comment:
We'll have to be a little careful about this merge as some of the gravity-ui-web files will have changed (such as this package.json) and I don't think this will get flagged up as a conflict (or it was, and it was resolved in the base package.json file).

https://github.com/buildit/gravity-ui-web/blob/develop/package.json

Reply:
It should be tracking the nested package.json to the one currently in `develop`'s root. I may have mis-merged when there was a conflict due to a large number of changes in the dependencies when I started tearing out pieces to go in the root (good catch). I've moved over the missing names.